assets/maps/area_definitions/test_area.onj

```diff
@@ -31,40 +25,26 @@ nodes: [
             currentlyBlocks: false,
             canBeStarted: true,
             isCompleted: false,
-            npc: 'my_cool_npc',
+            npc: 'Guy from Salem',
```

Comment:
generell am NPC concept, der "display name" sollte evtl. nicht gleich dem "name" sein (ist im NormalFall so, falls man den displayName später ändern will, ist mir aber relativ egal, ob wir das so lassen oder nicht)

Reply:
Havent gotten around to adding display names yet, but they are plannend